rocketpool-cli/network/dao-proposals.go

```diff
@@ -38,36 +39,32 @@ func getActiveDAOProposals(c *cli.Context) error {
 		return err
 	}
 
-	currentVotingDelegate, err := rp.GetCurrentVotingDelegate()
-	if err != nil {
-		return err
-	}
-
 	// Voting status
-	fmt.Printf("%s=== DAO Snapshot Voting ===%s\n", colorGreen, colorReset)
+	fmt.Printf("%s=== Snapshot Voting ===%s\n", colorGreen, colorReset)
 	blankAddress := common.Address{}
-	if snapshotProposalsResponse.VotingDelegate == blankAddress {
-		fmt.Println("The node does not currently have a voting delegate set, and will not be able to vote on Rocket Pool Snapshot governance proposals.")
+	if snapshotProposalsResponse.SignallingAddress == blankAddress {
+		fmt.Printf("The node does not currently have a snapshot signalling address set.\nTo learn more about snapshot signalling, please visit %s.\n", signallingAddressLink)
 	} else {
-		fmt.Printf("The node has a voting delegate of %s%s%s which can represent it when voting on Rocket Pool Snapshot governance proposals.\n", colorBlue, snapshotProposalsResponse.VotingDelegate.Hex(), colorReset)
+		fmt.Printf("The node has a signalling address of %s%s%s which can represent it when voting on Rocket Pool onchain governance proposals.", colorBlue, snapshotProposalsResponse.SignallingAddressFormatted, colorReset)
+		fmt.Println("")
 	}
 
 	voteCount := 0
-	for _, activeProposal := range snapshotProposalsResponse.ActiveSnapshotProposals {
-		for _, votedProposal := range snapshotProposalsResponse.ProposalVotes {
+	for _, activeProposal := range snapshotProposalsResponse.SnapshotResponse.ActiveSnapshotProposals {
+		for _, votedProposal := range snapshotProposalsResponse.SnapshotResponse.ProposalVotes {
```

Comment:
yikes, ok, remind me later, but we should modify snapshotProposalsResponse so it creates a dict instead of a list....

Reply:
more topically for this PR:
I see these double-nested for-loops in pdao/status.go as well. Can we break it out into a function that accepts a SnapshotResponse and returns the voteCount? Nice deduplication opportunity.

Actually, if we define SnapshotResponse, we can write a receiver function ie:
`func (s *SnapshotResponse) VoteCount() uint`